1.8/administration/categories-and-forums.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: page
+title: "Categories and Forums Management"
+categories: [administration]
+---
+
+## About Categories and Forums
+
+This is most basic guideline how to make and edit your first forums and categories at your forum.
+
+Categories and forums are root of your forum. It is suggested that they should be in some order so users and visitors can get good view of all categories and forums of your forum. There will be explained basic princip of how categories and forums work and what they are for.
```

Comment:
I think this should be: 

"Categories and forums are shown on the index of your forum. We suggest they are arranged in some order so users and visitors can get good view of all categories and forums on your website."